src/Symfony/Component/AssetMapper/ImportMap/ImportMapEntry.php

```diff
@@ -27,6 +30,12 @@ public function __construct(
         public readonly ?string $url = null,
         public readonly bool $isDownloaded = false,
         public readonly bool $preload = false,
+        public readonly string $type = self::TYPE_JS,
     ) {
     }
+
+    public static function getValidTypes(): array
+    {
+        return [self::TYPE_JS, self::TYPE_CSS];
```

Comment:
It's a nice idea, but there's a hiccup. When we dump the `importmap.php` file, we use `var_export()`. It will dump the full class name inline, which is quite ugly:

```
'file3.css' => [
    'path' => 'file3.css',
    'type' => \Symfony\Component\AssetMapper\ImportMap\ImportMapType::CSS,
    'preload' => true,
],
```

Reply:
the dumped file could be using the backing value, using `ImportMapType::from()` in the place where you currently use this `getValidTypes` to validate the content of the file being loaded. the dumped file does not contain exported ImportMapEntry objects.

Reply:
Done!